Packs/CommonScripts/Scripts/script-DeleteConext.yml

```diff
@@ -63,9 +69,11 @@ script: >
           if (isSubPlaybookKey) {
               key = 'subplaybook-${currentPlaybookID}.' + key;
           }
-          var result = executeCommand('delContext', {key: key});
-          if(!result || result.type === entryTypes.error) {
-              errorsStr = errorsStr + "\n" + result.Contents;
+          if (key !== "DBotScore" || !DBotScore) {
```

Comment:
When would the key be equal to DBotScore? Don't we delete it if it's there?

Reply:
We delete it from `keysToKeep`, this loop is on `fieldsToDelete`